### PR TITLE
[cmds] Cleanup in applets Makefiles

### PR DESCRIPTION
**Pull request by mfld-fr**
Removed unused parts of the Makefiles to make the room clear before improving the selection of the applets, as discussed on the mailing list: http://www.spinics.net/lists/linux-8086/msg00763.html

### CODE REVIEW

File: elkscmd/inet/tinyirc/Makefile [hunk 1]
```diff
@@ -67,4 +67,17 @@ tinyirc.o: tinyirc.c Makefile
 
 tinyirccv.o: tinyirccv.c Makefile
 	$(CC) $(CFLAGS) $(DEFINES) -c tinyirccv.c -o tinyirccv.o
+
```
Comment (mfld-fr) [inline elkscmd/inet/tinyirc/Makefile:70]: These ones are to fix error messages during the build of the image.

File: elkscmd/misc_utils/Makefile [hunk 2]
```diff
@@ -6,29 +6,7 @@ include $(BASEDIR)/Make.defs
...
-NET_USR_SBIN	= 
-
-STD_BIN 	= 
-STD_LIB 	= 
-STD_SBIN	= 
-STD_USR_BIN	= ed tar compress uncompress zcat miniterm
-STD_USR_LIB	= 
-STD_USR_MAN	= compress.1 uncompress.1 zcat.1
-STD_USR_SBIN	= 
+STD_USR_BIN    = ed tar compress uncompress zcat miniterm
```
Comment (mfld-fr) [inline elkscmd/misc_utils/Makefile:9]: Kept only in this file because used below.

Comment (jbruchon) [reply elkscmd/misc_utils/Makefile:9]: You can probably convert this to the PRGS variable used by most of the other applet directories.

Comment (mfld-fr) [reply elkscmd/misc_utils/Makefile:9]: Yes, will do it.

Comment (mfld-fr) [reply elkscmd/misc_utils/Makefile:9]: Done in commit 21b5217bab5f99caaf0ea13fdd8cf5547bd500b3.

**Pull request merged**